app/styles/components/c-footer.scss

```diff
@@ -7,54 +7,121 @@
   //Section Founding Logos on footer
   .logos-footer {
     border-bottom: 2px solid rgba($color-4, .2);
-    display: flex;
-    justify-content: space-between;
-    padding: 0 50px 55px 0;
+    display: block;
+    padding: 0 0 30px;
+
+    @media #{$mq-tablet} {
+      display: flex;
+      justify-content: space-between;
+      padding: 0 50px 55px 0;
+    }
 
     .partner-footer,
     .sponsor-footer {
       > .logo-list {
         align-items: center;
         display: flex;
+        list-style: none;
+      }
 
-        justify-content: space-between;
+      >.logo-list-funding {
+        display: block;
         list-style: none;
+        width: 100%;
+
+        li {
+          width: 100%;
+        }
+      }
+
+      .funding-text {
+        color: $color-16;
+        display: block;
+        font-size: $font-size-normal;
+        margin-bottom: 15px;
+
+        @media #{$mq-tablet} {
+          display: none;
+        }
+      }
+
+      .logo-item {
+        margin-right: 15px;
+
+        @media #{$mq-laptop} {
+          margin-right: 55px;
+        }
+
+        .logo-ldf {
+          width: calc(100%/3);
+
+          @media #{$mq-mobile-large} {
+            max-width: none;
```

Comment:
Is this necessary? You are not using `max-width` previously, there's no need to set it.